packages/dbs-ai/client/views/app/tabbar/smarti.js

```diff
@@ -38,13 +38,30 @@ Template.dbsAI_smarti.onRendered(function() {
 			const WEBSOCKET_URL =
 				`ws${ RocketChat.settings.get('Site_Url').substring(4) }websocket/`;
 
+			let customSuffix = RocketChat.settings.get('Assistify_AI_DBSearch_Suffix') || '';
+			customSuffix = customSuffix.replace(/\r\n|\r|\n/g, '');
 
-			self.smarti = new window.SmartiWidget(self.find('.external-message'), {
+			const smartiOptions = {
 				socketEndpoint: WEBSOCKET_URL,
 				smartiEndpoint: DBS_AI_Redlink_URL,
 				channel: self.data.rid,
-				inputCssSelector: '.autogrow-shadow'
-			});
+				inputCssSelector: '.autogrow-shadow',
+				widget:{
+					'query.dbsearch.keyword': {
```

Comment:
@ruKurz is `query.dbsearch.keyword` really needed? what is the semantic of the sub-property `keyword`? I thought it was a separate query provider.